### DISCUSSION TITLE
[Headless] Combobox setting to get selected value property

### DISCUSSION THREAD

**Discussion by coder2000** [alpinejs/alpine#3865] [3. Feature ideas]
When setting a field name to add a hidden field for a form it would be nice to supply an optional property to return from the value object, if it is one, instead of providing all the values.

Example:

The value for the combobox is something like {"id":1, "name":"Jerry"} and I would like to just get the id property back to the server.

**Comment by ekwoka** [comment]
Can you show some code? To clarify what you mean

**Reply by coder2000** [reply]
Something like the following is what I am thinking of. If no property is provided than the full object is handled as it is currently.

```
<div x-combobox name="framework" property="id">
  <input type="hidden" name="framework" value="1">
</div>
```
